### ISSUE TITLE
Remove redundant portions of "Create shipping label" error notifications

### ISSUE THREAD

**Issue by waclawjacek** [Automattic/woocommerce-services#2646]
Currently when the Connect Server returns an error, we display it like so:

`Error: The WooCommerce Shipping & Tax server returned: Bad Request Phone Number must be provided in the 'Origin address' tab. (400)`

This seems sloppy to me and as if someone didn't take the time to polish the format.

* It is obvious that it's an error (there will be the red color and a `(!)` symbol) but we can keep it.
* It is obvious it's coming from the WooCommerce Shipping & Tax extension because it happens after you interact with this extension.
* `Bad Request` and the trailing `(400)` are more an HTTP detail and since we have a more specific error message ("Phone Number must be provided in the 'Origin address' tab"), we don't need the `Bad Request`/`400` bits.

My suggestion is to replace this message (and any other that has a user-friendly explanation) with something like:

`Phone Number must be provided in the 'Origin address' tab.`

WDYT, @j111q & @Automattic/escargot?

**Comment by samnajian** [comment]
@waclawjacek I wanna add that when user clicks on `Verify Address`, it should verify the phone number, currently it's showing an error message when the user is already in the last step and basically forces the user go to back to the `Origin Address` which is really annoying and miss-leading given that I already have a green checkmark for my `Origin Address` section.

**Comment by waclawjacek** [comment]
Yes it is! This is something that we implemented as a backward-compatible solution to this problem, with the intention of rolling out a better solution later. Could you please create a separate issue for this?

**Comment by samnajian** [comment]
Separate [Issue](https://github.com/Automattic/woocommerce-services/issues/2660) is created.

**Comment by Abdalsalaam** [comment]
Shipping related work now happens in WooCommerce Shipping and we recommend using it: https://wordpress.org/plugins/woocommerce-shipping/
